arkane/ess/psi4.py

```diff
@@ -72,8 +70,8 @@ def get_number_of_atoms(self):
             while line != '' and n_atoms == 0:
                 # Automatically determine the number of atoms
                 if 'Center              X                  Y                   Z               Mass ' in line and n_atoms == 0:
```

Comment:
Is the `n_atoms` condition needed in both line 70 and line 72?

Reply:
No. I fixed it.